base/float16.jl

```diff
@@ -149,8 +149,16 @@ for func in (:sin,:cos,:tan,:asin,:acos,:atan,:sinh,:cosh,:tanh,:asinh,:acosh,
         $func(a::Complex32) = complex32($func(complex64(a)))
     end
 end
-atan2(y::Float16, x::Float16) = float16(atan2(float32(y), float32(x)))
-hypot(a::Float16, b::Float16) = float16(hypot(float32(a), float32(b)))
+
+for func in (:div,:fld,:cld,:rem,:mod,:mod1,:rem1,:fld1,
```

Comment:
I added a test for the immediate issue. In general, `Float16` is pretty spares when it comes to tests, but perhaps that's a separate issue.